src/deepsparse/image_classification/constants.py

```diff
@@ -16,7 +16,1011 @@
 __all__ = [
     "IMAGENET_RGB_MEANS",
     "IMAGENET_RGB_STDS",
+    "IMAGENET_LABELS",
 ]
 
 IMAGENET_RGB_MEANS = [0.485, 0.456, 0.406]
 IMAGENET_RGB_STDS = [0.229, 0.224, 0.225]
+
+IMAGENET_LABELS = [
```

Comment:
was wondering why this PR was so long :)

Reply:
I was really unhappy about putting those 1000 lines of code effectively, but I guess this is not something super awful.